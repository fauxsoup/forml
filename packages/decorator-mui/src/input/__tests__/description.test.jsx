import Description from "../description";
import Context from "@forml/context";
import React from "react";
import ReactPDF from "@react-pdf/renderer";
import { render } from "@testing-library/react";
import * as decorator from "../../";

describe("renders", function() {
    let form;

    beforeEach(function() {
        form = { type: "fieldset", description: "description" };
    });

    test("with description", function() {
        const { container } = render(
            <Context.Provider value={{ decorator }}>
                <Description form={form}>{form.description}</Description>
            </Context.Provider>
        );

        expect(container).toMatchSnapshot();
        expect(container.textContent).toBe('description');
    });
});
