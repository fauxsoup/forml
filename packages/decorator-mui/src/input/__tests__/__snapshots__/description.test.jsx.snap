// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders with description 1`] = `
<div>
  <p
    class="MuiFormHelperText-root css-1d1r5q-MuiFormHelperText-root"
  >
    description
  </p>
</div>
`;
